msg/ItemizedStats.msg

```diff
@@ -1,7 +1,7 @@
-# This message contains itemized statiistics for PackML statistics reporting
+# This message contains itemized statistics for PackML statistics reporting
 # Each item containds a unique ID that defines the "item".  
 # Example usage includes, error and warning events.
 
 int16 id                          # unique identifier (user defined)
-int32 count                       # number of occurances
+float32 count                     # total value of itemized stat
```

Comment:
If count has changed meaning from total occurances of the stat to the total value of the stat, should the name change as well (i.e. to "total_value"?)

Reply:
I wanted to do a least harm approach here, but you have a point. Existing applications could coerce an int to this field and operate normally, while changing the field name would require applications to update.